Reports/progress_report.md

```diff
@@ -0,0 +1,292 @@
+    - DBeaver drivers were not installing correctly during VM building, so it tries to download them from the internet.
+      No problem on T1, but fails on T2.
+- Factoring storage creation and account deployments out of main deployment script now allows for a more resilient process (not having to re-run everything when one fails)
+- MS changed the name of Azure Directory to Microsoft Entra ID which made necessary to spend time updating documentation and code
+- Factor SHM storage creation out of main deployment script: PR [#1673](https://github.com/alan-turing-institute/data-safe-haven/pull/1673)
+- Add all contributors table to project README and docs: PR[#1649](https://github.com/alan-turing-institute/data-safe-haven/pull/1649).
+- Removal of MSRDS (PR: [#1535](https://github.com/alan-turing-institute/data-safe-haven/pull/1535)) which reduces support burden and codebase complexity, instead Guacamole implementation is more robust and secure.
+- Removal of CoCalc (PR: [#1554](https://github.com/alan-turing-institute/data-safe-haven/pull/1554) Reduces support burden for future releases by removing a largely unused feature.
+- Drop Microsoft Remote Desktop: (PR: [#1159](https://github.com/alan-turing-institute/data-safe-haven/issues/1159) primarily for increased security as it shows more issues than Guacamole, in doing this several other open issues are resolved.
+- Documentation improvements and updates
+
+### Identify and implement core IAC changes: [#28](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+
+#### Goal
+Make DSH deployment more robust and development easier through using IAC and configuration management.
+- Take advantage of IAC and configuration management in the DSH codebase which will
+  - Make deployments faster
+  - Make deployments more reliable
+  - Make development easier
+- Move away from non-idempotent, bespoke scripts (Powershell, bash, cloud-init)
+
+##### Definition of done
+On the release of a new major version which removes legacy, script-based deployment.
+
+#### Progress
+Arrived at a IAC MVP version of the code, available as a penetration tested [pre-release](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1).
+
+This new code is better and easier for users to deploy, however some incompatibilities with the old code would require extensive work. 
+As PowerShell heads to its final release it was decided not to work on fixing these.
+
+Since finishing migration a lot of the work is focusing on structuring the code, small improvements aimed at the user experience and robustness.
+
+These are main references and milestones, a more complete list is available on the [story issue](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+- Codebase pre-release https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1
+- Next version milestone https://github.com/alan-turing-institute/data-safe-haven/milestone/20
```

Comment:
I need to sort out the milestones to reflect the new thinking w.r.t sprints aligned with DSGs.